pkgs/top-level/python-packages.nix

```
@@ -3996,6 +4006,10 @@ in {
   pylint = if isPy3k then callPackage ../development/python-modules/pylint { }
            else callPackage ../development/python-modules/pylint/1.9.nix { };
 
+  pylint-django = disabledIf isPy27 (callPackage ../development/python-modules/pylint-django { });
```

I think it's preferred to move disable logic into the nix fix
```suggestion
  pylint-django = callPackage ../development/python-modules/pylint-django { };
```